src/core/generator/generator.rs

```diff
@@ -95,7 +95,9 @@ impl Generator {
                     .try_fold(ConfigModule::default(), |config_module, path| {
                         let content = std::fs::read_to_string(path.as_ref())?;
                         let converter = OpenApiToConfigConverter::new(query, content)?;
-                        let config = converter.convert();
+                        let config = converter
+                            .convert()
+                            .ok_or(anyhow::anyhow!("Failed to convert"))?;
```

Comment:
I think this shouldn't fail, it should return default with logs whatever is not possible to convert.

Reply:
made the suggested changes